app/models/subnet/ipv4.rb

```diff
@@ -0,0 +1,42 @@
+require 'socket'
+
+class Subnet::Ipv4 < Subnet
+  def self.default_sti_class
+    'Subnet::Ipv4'
```

Comment:
I agree, that's a leftover from the IPv6 branch and can safely be deleted from the IPv4 class.

The code currently has the problem, that when you create a new instance of Subnet::Ipv6 without a type provided, it get's casted to Subnet::Ipv4 because ```self.default_sti_class``` is inherited to Subnet::Ipv6 as "Subnet::Ipv4". Let me see, what can be done against this...